react/UploadQueue/index.jsx

```diff
@@ -64,20 +64,26 @@ const Pending = translate()(props => (
   </Typography>
 ))
 
-const formatRemainingTime = durationInSec => {
+export const formatRemainingTime = durationInSec => {
   const later = Date.now() + durationInSec * 1000
-  return formatDistanceToNow(later)
+  return formatLocallyDistanceToNow(later)
 }
 
 const RemainingTime = ({ durationInSec }) => {
   const { t } = useI18n()
+
   return (
     <Typography
       variant="caption"
       className={cx(styles['upload-queue__progress-caption'], 'u-ellipsis')}
     >
       {t('item.remainingTime', {
-        time: formatRemainingTime(durationInSec)
+        time: formatRemainingTime(durationInSec),
+        smart_count:
+          // https://date-fns.org/v2.28.0/docs/formatDistanceToNow
+          durationInSec < 90 || (durationInSec > 2670 && durationInSec < 5370)
+            ? 1
+            : 2
```

Comment:
> pourquoi y'a que 1 et 2 en valeur possible ? Je dois pas bien comprends ce qu'on chercher à faire ici 🤔 c'est pas le nombre de minutes restantes ? Si oui, on peut pas avoir plus de 2 minutes ?
Not at all.
If smart_count get 0 or 1, it will use the singular form. If smart_count get a number more than 1, it will use plural form.


> nit: on pourrait sortir ça dans une méthode avec un nom explicite, ça aiderait à la compréhension
renamed into `numberOfReferencesForPluralForm` + fully tested.